go.mod

```diff
@@ -23,11 +25,14 @@ require (
 	github.com/spf13/cobra v1.1.3
 	github.com/spf13/viper v1.7.1
 	github.com/stretchr/testify v1.7.0
+	github.com/thoas/go-funk v0.8.1-0.20210502090430-efae847b30ab
 	github.com/vbauerster/mpb/v6 v6.0.3
 	github.com/zclconf/go-cty v1.8.3
 	golang.org/x/crypto v0.0.0-20210506145944-38f3c27a63bf
 	golang.org/x/sync v0.0.0-20210220032951-036812b2e83c
 	golang.org/x/term v0.0.0-20201126162022-7de9c90e9dd1
 	gopkg.in/natefinch/lumberjack.v2 v2.0.0
-	gopkg.in/yaml.v3 v3.0.0-20210107192922-496545a6307b
 )
+
+// TODO: remove after approval
```

Comment:
> I'm not quite sure right now what the issue is but I compiled this PR locally (including the cq-provider-sdk pr) and when I run it I get the following error:
> 
> `ERROR: column \"id\" of relation \"aws_waf_rule_groups\" does not exist (SQLSTATE 42703)`
> 
> Is this expected? Do we have to change something on the provider side?

Yes, we changed "id" to "cq_id" this breaks all previous tables and requires us to drop previous version schema, this might also break some of the policy queries.

Reply:
I ran this against a fresh database. Should this behave differently?